src/app/home/page.tsx

```diff
@@ -87,26 +75,26 @@ export default function Index() {
           </div>
           <div className="w-1/4 border-l border-gray-300 bg-white p-4 pl-8">
             <h1 className="mb-4 text-xl font-bold">Recently Viewed Events</h1>
-            <Suspense fallback={<p>Loading...</p>}>
-              {!isLoading && upcomingEvents.length === 0 ? (
-                <p>No recently viewed events.</p>
-              ) : (
-                upcomingEvents.map((event) => (
-                  <div key={event.id}>
-                    <EventCard
-                      eventId={event.id}
-                      title={event.title}
-                      starttime={event.starttime}
-                      endtime={event.endtime}
-                      date={null}
-                      days={null}
-                      key={event.id}
-                    />
-                    <div className="mb-4"></div>
-                  </div>
-                ))
-              )}
-            </Suspense>
+            {isLoading ? (
+              <p>Loading...</p>
```

Comment:
Thoughts on "Loading" dots centered in middle of the page instead? https://daisyui.com/components/loading/